benchmark/single-source/SortStrings.swift

```diff
@@ -1520,7 +1520,7 @@ let unicodeWords: [String] = [
   "❄️physicianless",
   "❄️bronchopathy",
   "❄️marabuto",
-  "❄️proprietory",
+  "❄proprietary",
```

Comment:
```suggestion
  "❄proprietory",
```

Reply:
It's still missing the variation selector unicode character prefix. The original version was correct.

Reply:
Those are suggestion for reverting to original version!

Reply:
It doesn’t look correct though. If you look at the original, it has a variation selector (it looks like an empty box) as the first character.

Reply:
Can you please create suggestion for it?

Reply:
```diff
-  "❄proprietary",
+  "❄️️️proprietary",
```

Reply:
If it's all the same, I'd just revert out these test changes.  Even if we're trying to err on the side of prescriptivism, "[proprietory](https://www.merriam-webster.com/dictionary/proprietory)" is a valid, if baroque, spelling.

Reply:
Now I reverted the file to original from the **master**!